src/calendar/index.js

```diff
@@ -77,14 +77,16 @@ class Calendar extends Component {
     /** Handler which gets executed when press arrow icon left. It receive a callback can go next month */
     onPressArrowRight: PropTypes.func,
     /** Style passed to the header */
-    headerStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.number, PropTypes.array])
+    headerStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.number, PropTypes.array]),
+    /** Provide heading level for proper accessibility when used with web (react-native-web) */
+    headingLevel: PropTypes.number
```

Comment:
Why not call the prop `ariaLevel` to make it more explicit?

Reply:
@Inbal-Tish thank you for the review.  I'd gladly make the change if there was a hope for this getting merged.  Let me know.

Reply:
I went ahead and updated the PR and made the change since it seems like the project is active again.  Thanks!